src/main/java/frc/robot/Constants.java

```diff
@@ -100,8 +103,7 @@ public class Auto {
     public class Shooter {
         public static final double SHOOT_SPEED = 1700;
         public static final double AMP_SHOOT_SPEED = SHOOT_SPEED * 0.4;
-        public static final double LONG_SHOT_SPEED = 5000;
-        public static final double AUTO_SHOT_SPEED = 3500;
+        public static final double LONG_SHOT_SPEED = 3500;
```

Comment:
I'm not worried about changing this unless we like, completely refactor and redo the robot code, but we shouldn't have handled it this way and instead based debounce time on, maybe the magnitude of move that was taking place